crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -578,8 +603,8 @@ impl IndexerReader {
                 .filter(objects::dsl::owner_id.eq(address.to_vec()))
                 .limit(limit as i64)
                 .into_boxed();
-            if let Some(object_type) = object_type {
-                query = query.filter(objects::dsl::object_type.eq(object_type));
+            if let Some(object_types) = object_types {
+                query = query.filter(objects::dsl::object_type.eq_any(object_types));
```

Comment:
add an index for object_type here?
https://github.com/MystenLabs/sui/blob/main/crates/sui-indexer/migrations_v2/2023-08-19-044023_objects/up.sql#L35